docs/rhoas/iac_tools_rhoas/README.adoc

```diff
@@ -0,0 +1,686 @@
+
+* Kafka instance
+* Service accounts
+* Topics
+* ACLS
+
+The procedure in this section shows how to create and run an example playbook. The example playbook includes modules for creating a Kafka instance, topic, and service account, and assigning permissions to the service account. You also configure modules to delete the resources that you previously created.
+
+.Prerequisites
+* You have a Red Hat account.
+* You have an offline token used to authenticate the Ansible modules with the {product-long-rhoas} API.
+
+[NOTE]
+The offline token is a refresh token with no expiry and can be used by non-interactive processes to provide an access token for {product-long-rhoas} to authenticate the user. The token is an OpenShift offline token and you can find it at https://cloud.redhat.com/openshift/token.
+
+
+.Procedure
+. In your IDE, create a new enviornment variables (`.env`) file.
+. In the `.env` file, add the following variables required by the Ansible collection:
++
+`API_BASE_HOST`:: This is the base URL for the API. For example, `\https://api.openshift.com`.
+`SSO_BASE_HOST`:: This is the base URL for the Red Hat single sign-on (SSO) service. For example, `\https://sso.redhat.com/auth/realms/redhat-external`.
+. Save the file in the root directory of the Ansible collection.
++
+NOTE: If you do not explicitly define the environment variables, the collection uses the URLs shown in the preceding step.
+
+. Open a terminal window.
+. Run a module in the terminal to perform a single task. For example, the following command shows how to run a module that creates a Kafka instance. Replace _<OFFLINE_TOKEN>_ with your OpenShift offline token.
++
+.Example create_kafka module
+[source,shell]
+----
+$ ansible localhost -m rhoas.rhoas.create_kafka -a 'name=unique-kafka-name billing_model=standard cloud_provider=aws plan="developer.x1" region="us-east-1" openshift_offline_token=<OFFLINE_TOKEN>'
+----
+
++
+Ansible runs the `rhoas.rhoas.create_kafka` task in the terminal and creates the instance.
+. To open the example playbook used in later steps, use your IDE to navigate to the `rhoas` folder in the `.ansible` directory.
++
+[source]
+----
+$ cd rhoas
+----
+
+. Open the `rhoas_test` example playbook. You can also open a web browser and navigate to https://github.com/redhat-developer/app-services-ansible/blob/main/rhoas_test.yml to open the `rhoas_test` playbook.
+. Inspect the contents of the example playbook. In particular, observe that the playbook has modules for certain tasks. You will use these modules to create a new playbook that performs the following tasks:
++
+* Create and delete a Kafka instance
+* Create and delete a service account
+* Create Access Control List (ACL) permission bindings
+* Create, update, and delete a topic
```

Comment:
Per https://github.com/redhat-developer/app-services-guides/pull/601/files#r1029132440, I think these should be gerund phrases.